datadog_checks_downloader/tox.ini

```diff
@@ -12,4 +12,4 @@ deps =
   -rrequirements-dev.txt
 commands =
   pip install -r requirements.in
-  pytest -vs --log-cli-level=debug
+  pytest -v --junit-xml=junit/test-{envname}.xml {posargs} --capture=no --log-cli-level=debug
```

Comment:
what does `--capture=no` do?

Reply:
```
  --capture=method      per-test capturing method: one of fd|sys|no.
  -s                    shortcut for --capture=no.
```

It avoid pytest to capture the stdout if the tests themselves are printing to stdout.

I just replace `-s` by `--capture=no` to make it more readable. No behavior change in this PR.

---

I'm not sure, why we wanted it to be like this. Maybe for debugging.  https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=2144&view=logs&j=1e9c9f74-2d69-5fdc-e1d4-ad8b36d1f0fb

@trishankatdatadog Can we remove `--capture=no` aka `-s` ?